src/modules/redesign/parsers/verband/gebauede.ts

```diff
@@ -37,7 +37,7 @@ export default <RedesignParser<VerbandGebaeudeWindow>>(({
         ...verbandParser({ doc, getIdFromEl }),
         buildings: Array.from(
             markerScript.matchAll(
-                /(?<=L\.marker\(\[)(?<lat>-?\d+(?:\.\d+)?)\s*,\s*(?<long>-?\d+(?:\.\d+)?)\](?:.|\n)*?(?<=\/buildings\/)(?<id>\d+)/gu
+                /(?<=L\.marker\(\[)(?<lat>-?\d+(?:\.\d+)?)\s*,\s*(?<long>-?\d+(?:\.\d+)?)\](?:.|\n)+?(?<=\/buildings\/)(?<id>\d+)/gu
```

Comment:
Is there a case where the previous failed? Currently I cant imagine a case where this change makes a difference.